### DISCUSSION TITLE
Missing Observations

### DISCUSSION THREAD

**Discussion by TimPasquini** [weather-gov/api#570] [General]
I'm using the stations/{station_id}/observations call to get weather data for a few stations and it seems like there are hours being randomly dropped fairly frequently.

For example, in KALB, using https://api.weather.gov/stations/KALB/observations?start=2022-07-25T23%3A51%3A00%2B00%3A00 to get everything since UTC midnight on the 25th to now, I found that 2022-07-27T00:51:00+00:00 is completely missing. The data that exists jumps from 23:51 on the 26th to 01:51 on the 27th.

Are the weather stations this unreliable or is the data being filtered out somehow? I'm trying to migrate off a paid API and that seems to return data for the missing hours at the same stations, not sure if they are fudging numbers or if NWS is just not returning everything.

**Comment by akrherz** [comment]
FWIW, the 27 0051z ob did flow over noaaport
```text
SAUS80 KWBC 270100
METAR
...snipped..
KALB 270051Z 27005KT 10SM FEW060 FEW200 22/13 A2997 RMK AO2 SLP146
     T02170128=

```

**Comment by scadergit** [comment]
This is a known issue that will be resolved in the next release later this year (NCO bundles issues into larger deploys).

**Reply by TimPasquini** [reply]
Excellent to hear, thank you!

**Reply by TimPasquini** [reply]
Just following up on this. Was there an update/release? I still get gaps in my data.

**Reply by sullynole** [reply]
The release with the fix will be early 2023, likely Feb-Mar.

**Comment by TimPasquini** [comment]
Was curious if anything was ever done about this. Also, it looks like KALB stopped updating at all at yesterday.
